src/Cairo.jl

```diff
@@ -13,11 +16,11 @@ Base.@deprecate_binding _jl_libpango      Cairo.libpango   false
 Base.@deprecate_binding _jl_libpangocairo Cairo.libpango   false
 
 function __init__()
-    check_deps()
-    # On Linux, FreeBSD and macOS we use FontConfig. Set FONTCONFIG_FILE to the
-    # config file we provide.
-    if !Sys.iswindows()
-        ENV["FONTCONFIG_FILE"] = joinpath(dirname(libcairo), "..", "etc", "fonts", "fonts.conf")
+    # On Linux and FreeBSD we use FontConfig. Set FONTCONFIG_PATH only if none
+    # of FONTCONFIG_PATH or FONTCONFIG_FILE is set.
+    if !(Sys.isapple() || Sys.iswindows()) && get(ENV, "FONTCONFIG_PATH", "") == "" &&
```

Comment:
Fontconfig is now on all platforms, right?

Reply:
Mmmm, yes, I believe that is true yes.